plugins/app.installer

```diff
@@ -18,21 +18,7 @@ build() {
         git fetch --all
         git checkout -q "${COMMIT}"
         # There's no real versions yet, but this'll just leave it blank
-        make bin/docker-app
-    )
-}
-
-build_mac() {
-    if [ ! -d "${DEST}" ]; then
-        git clone "${REPO}" "${DEST}"
-    fi
-    (
-        cd "${DEST}"
-        git fetch --all
-        git checkout -q "${COMMIT}"
-        make bin/docker-app-darwin
-        # Move it so that `install_plugin` still works as expected
-        mv bin/docker-app-darwin bin/docker-app
+        make dynamic
```

Comment:
The Windows build creates `docker-app` binary without the `.exe` extension

The resulting zip file has contains these files:

```
$ tar tvf static/build/win/docker-0.0.0-20190318164418-36d2c8b48e.zip 
drwxrwxrwx  0 0      0           0 Mar 19 07:13 docker/
-rwxrwxrwx  0 0      0    60088853 Mar 19 07:13 docker/dockerd.exe
-rwxrwxrwx  0 0      0    53977859 Mar 19 07:13 docker/docker.exe
-rwxrwxrwx  0 0      0    35936768 Mar 19 07:13 docker/docker-app
```